modules/KalturaSupport/resources/mw.KBasePlugin.js

```diff
@@ -163,7 +163,7 @@ mw.KBasePlugin = Class.extend({
 
 		return false;
 	},
-	bind: function( eventName, callback ){
+	bind: function( eventName, callback , once ){
```

Comment:
@itaykinnrot wouldn't it be better to add just bindOnce, more expressive then passing true in function call